deployment/aks-periscope.yaml

```diff
@@ -23,6 +23,18 @@ rules:
 - apiGroups: ["admissionregistration.k8s.io"]
   resources: ["mutatingwebhookconfigurations", "validatingwebhookconfigurations"]
   verbs: ["get", "list", "watch"]
+- apiGroups: ["apiextensions.k8s.io"]
+  resources: ["customresourcedefinitions"]
+  verbs: ["get", "list", "watch"]
+- apiGroups: ["split.smi-spec.io"]
+  resources: ["trafficsplits"]
+  verbs: ["get", "list", "watch"]
+- apiGroups: ["access.smi-spec.io"]
+  resources: ["traffictargets"]
+  verbs: ["get", "list", "watch"]
+- apiGroups: ["specs.smi-spec.io"]
+  resources: ["httproutegroups", "tcproutes", "udproutes"]
+  verbs: ["get", "list", "watch"]
```

Comment:
These kubernetes rbac `ClusterRole` configs allow the role to access resources within the SMI api groups (`*.smi-spec.io`).

Is there a way to specify rbac api group access using wildcards? I'm trying to achieve the following in lieu of what I've written above, but I do not know if this is possible (or even what the syntax should be). Help appreciated.

```yaml
- apiGroups: ["*.smi-spec.io"]
  resources: ["*"]
  verbs: ["get", "list", "watch"]
```

For context on the SMI crds that I am interested in, this is the output when I run `kubectl api-resources | grep smi-spec.io`:

```tsv
NAME                              SHORTNAMES   APIVERSION                          NAMESPACED   KIND
traffictargets                    tt           access.smi-spec.io/v1alpha3         true         TrafficTarget
httproutegroups                   htr          specs.smi-spec.io/v1alpha4          true         HTTPRouteGroup
tcproutes                         tr           specs.smi-spec.io/v1alpha4          true         TCPRoute
udproutes                         ur           specs.smi-spec.io/v1alpha4          true         UDPRoute
trafficsplits                     ts           split.smi-spec.io/v1alpha2          true         TrafficSplit
```

Reply:
Not for apiGroups, according to this: https://github.com/kubernetes/kubernetes/blob/a55bd631728590045b51a4f65bba31aed1415571/staging/src/k8s.io/api/rbac/v1beta1/types.go#L53-L56
You can try: 
```yaml
- apiGroups: ["access.smi-spec.io", "specs.smi-spec.io"]
  resources: ["*"]
  verbs: ["get", "list", "watch"]
```

Reply:
Thanks Rita!